Doc/whatsnew/3.8.rst

```diff
@@ -1502,6 +1505,11 @@ CPython bytecode changes
   when awaiting a next item in an :keyword:`async for` loop.
   (Contributed by Serhiy Storchaka in :issue:`33041`.)
 
+* The :opcode:`MAP_ADD` now expects the value as the first element in the
+  stack and the key as the second element. This change was made so the key
+  is always evaluated before the value in dictionary comprehensions, as
+  porposed by :pep:`572`. (Contributed by Jörn Heissler in :issue:`35224`.)
```

Comment:
porposed -> proposed

Reply:
Fixed in https://github.com/python/cpython/pull/14371